### PR TITLE
Update Italian localization

### PR DESCRIPTION
**Pull request by M0Rf30**
As in the title
Regards

### CODE REVIEW

**Review by AJenbo — APPROVED** [review]

File: Translations/it.po [hunk 1]
```diff
@@ -3610,680 +3598,676 @@ msgstr "Amuleto del Chierico"
...
+msgstr "{0} di {1}"
 
 #. TRANSLATORS: Constructs item names. Format: {Prefix} {Item} of {Spell}. Example: King's War Staff of Firewall
-#: Source/items.cpp:1193
-#, fuzzy
+#: Source/items.cpp:1192
 msgctxt "spell"
 msgid "{0} {1} of {2}"
-msgstr "{:s} {:s} di {:s}"
+msgstr "{1} {0} di {2}"
```
Comment (AJenbo) [inline Translations/it.po:3639]: glad to see this was useful :)

Comment (M0Rf30) [reply Translations/it.po:3639]: Yes. Have to refine some things. Italian also have grammatical gender for prepositions. an example:
the arrival of the train - train is male so of = del (l'arrivo del treno)
the doors of perception - perception is female so of = della (le porte della percezione).
I'm trying to adapt wherever is possible. But have to deeply dive into testing

Comment (AJenbo) [reply Translations/it.po:3639]: im currently working on a solution for grammatical genders

Comment (AJenbo) [reply Translations/it.po:3639]: You could make the prepositions part of the suffix

`{1} {0} {2}`

```
di ferro
rossa
d'angelo
del soldato
```

```
Ascia
Scudo
```

```
del massacro
di deviazione
della fragilità
dei giganti
dello stolto
della luna
delle stelle
dell'armonia
```

I know this dosent solve the issue of gendered prefixes, but I think it's the best you can currently do.

**Pull request merged**